src/components/pages/guide-page/GuidePage.tsx

```diff
@@ -3,12 +3,14 @@ import { ComponentMapper } from '../../ComponentMapper';
 import { GuidePageProps } from '../../../types/content-props/dynamic-page-props';
 import { ThemedPageHeader } from '../../_common/headers/themed-page-header/ThemedPageHeader';
 import { BEM } from '../../../utils/classnames';
+import { PreviewWarning } from 'components/_common/previewWarning/PreviewWarning';
 
 const bem = BEM('product-page');
 
 export const GuidePage = (props: GuidePageProps) => {
     return (
         <div className={bem()}>
+            <PreviewWarning />
```

Comment:
Kan denne settes inn i f.eks PageWrapper-komponenten? Potensielt kan det være behov for den på alle innholdstyper.

Reply:
Ja sant, det... Var litt usikker på hvor dypt i hierarkiet jeg burde legge den. Fikset!